src/components/molecules/ScheduleItemNG/ScheduleItemNG.tsx

```diff
@@ -74,31 +71,29 @@ export const ScheduleItemNG: React.FC<ScheduleItemNGProps> = ({
     differenceInCalendarDays(eventEndTime(event), eventStartTime(event))
   );
   const isCurrentEventLive = isEventLive(event);
-  const roomUrlParam = getUrlParamFromString(event.room ?? "");
 
   const handleCopyEventLink = useCallback(
     (e?: React.MouseEvent<HTMLButtonElement>) => {
+      // @debt get rid of stopPropagation() in the project allowing a valid event bubbling
       e && e.stopPropagation();
 
-      const eventLink = getFullVenueInsideUrl(roomUrlParam);
+      const eventLink = eventRoom
+        ? eventRoom.url
+        : getFullVenueInsideUrl(eventVenue?.id ?? "");
```

Comment:
Same logic as the following?

```suggestion
      const eventLink = eventRoom?.url ?? getFullVenueInsideUrl(eventVenue?.id ?? "");
```

Reply:
yes, looks like it does the same. fixed: https://github.com/sparkletown/sparkle/pull/2183/commits/2be26c4e66c5595df4c893f818cc364b297a21b3